app/helpers/application_helper.rb

```diff
@@ -319,6 +319,9 @@ def db_to_controller(db, action = "show")
     when "User", "Group", "Patch", "GuestApplication"
       controller = "vm"
       action = @lastaction
+    when "Host" && action == 'x_show'
```

Comment:
This condition is wrong (it's `when true` or `when false`, won't ever match db).

@lgalis Do you remember what this was for?

I assume it was meant to be 
```
when "Host"
  if action == 'x_show'
     controller = "infra_networking"
     action = @lastaction
  end
```

but what happens for other actions?